pyqttoolkit/services/file_dialog.py

```diff
@@ -49,5 +49,7 @@ def _get_default_directory(self):
             if default_location and path.isdir(default_location):
                 return default_location
             return QStandardPaths.writableLocation(QStandardPaths.DocumentsLocation)
+        elif self._project_manager.filename == '***Recovered***':
```

Comment:
Perhaps instead we check if `self._project_manager.filename and not_a_dir(dirname(self._project_maanager.filename))`